content/developers/developer-patterns/3rdparty-verification/index.md

```diff
@@ -161,39 +144,37 @@ This command will describe the activity for all tenants that have recently added
   {{< /tab >}}
 {{< /tabs >}}
 
-## Replicating the logs for the tenants with activity
-
-To automatically replicate the logs that changes are detected for pipe the output of `watch` into `replicate-logs`
+## Replicating the Logs for the Tenants With Activity
 
-{{< tabs >}}
-   {{< tab name="bash" >}}
+To automatically replicate the logs that changes are detected for, pipe the output of `watch` into `replicate-logs`
 
-   ```bash
-   veracity watch \
-   | veracity \
-       replicate-logs --replicadir merklelogs
-   ```
+- Replicate the logs for all tenants that have been active in the default time horizon
 
-   This will replicate the logs for all tenants that have been active in the default time horizon
+  {{< tabs >}}
+    {{< tab name="bash" >}}
 
-  {{< /tab >}}
-{{< /tabs >}}
+  ```bash
+  veracity watch | \
+    veracity \
+    replicate-logs --replicadir merklelogs
+  ```
 
-{{< note>}} Take care with larger time horizons, you may run into issues with rate limiting.{{< /note >}}
+    {{< /tab >}}
+  {{< /tabs >}}
 
-## Replicating just the latest changes to your log
+{{< note>}} Take care with larger time horizons, it may trigger issues with rate limiting.{{< /note >}}
 
-By default, your full tenant log is replicated. The storage requirements are
```

Comment:
I'm happy to remove this, but do we not need to cover the local storage requirements some where ? Maybe the command line help is a better place

Reply:
ah you just moved it!